device/soc_descriptors/blackhole_140_arch_no_eth.yaml

```diff
@@ -0,0 +1,71 @@
+# Note taken from software repo - may need updates.
+grid:
+  x_size: 17
+  y_size: 12
+
+arc:
+  [ 8-0 ]
+
+pcie:
+  [ 11-0 ]
```

Comment:
Heads up on https://github.com/tenstorrent/tt-umd/issues/279

Reply:
Assigned myself, I will address it in a separate PR